src/EditorFeatures/Core/InheritanceMargin/AbstractInheritanceMarginService.cs

```diff
@@ -1,107 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this was moved down to Features.  Unfortunatelly, github seems to view it as a delete :(